test/unittests/end_to_end_test.cpp

```diff
@@ -33,7 +33,8 @@ TEST(end_to_end, milestone1)
     const auto module = parse(wasm);
     auto instance = instantiate(module);
 
-    EXPECT_THAT(execute(*instance, 0, {20, 22}), Result(20 + 22 + 20));
```

Comment:
This wouldn't work with span?

Reply:
No, because `std::span` is a reference type and `{20, 22}` is a temporary object.

Reply:
So in the "external API" we should have a wrapper then.

Reply:
Yes, we can have this in public API. But, I'm not convinced this is super useful. In normal usage you rarely pass constant args to wasm. They come usually from other sources like CLI, etc.

For now I placed this in `fizzy::test`.